pdf-bygger/src/main/kotlin/no/nav/pensjon/brev/pdfbygger/latex/LatexCompileService.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.pensjon.brev.pdfbygger.latex
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.withContext
+import kotlinx.coroutines.withTimeoutOrNull
+import no.nav.brev.brevbaker.PDFCompilationOutput
+import no.nav.pensjon.brev.pdfbygger.PDFCompilationResponse
+import no.nav.pensjon.brev.template.render.DocumentFile
+import org.slf4j.LoggerFactory
+import java.io.File
+import java.io.IOException
+import java.nio.file.Path
+import kotlin.io.path.createTempDirectory
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.milliseconds
+
+private const val COMPILATION_RUNS = 2
+
+class LatexCompileService(
```

Comment:
Er det meininga at denne er i tillegg til `no.nav.pensjon.brev.pdfbygger.LaTeXService`?

Reply:
Beholdt den litt imens jeg holdt på med en merge conflict for å kopiere over kode-snutter jeg trengte fra endringene. Var ikke meningen å beholde den.